static-site/variables.tf

```diff
@@ -1,31 +1,28 @@
-# aws region
+// aws region
 provider "aws" {
   region = "us-east-1"
 }
 
 // root domain name
-variable "root_domain_name" {
-  default = "digital.mass.gov"
+variable "zone_id" {
+  type        = "string"
+  description = "The zone that domain will be added to."
 }
 
-// new statis sites are subdomains under *.digital.mass.gov
-variable "sub_domain_name" {
+// new site domain name
+variable "domain_name" {
   type        = "string"
-  description = "The full sub domain name"
+  description = "The full domain name being added."
 }
 
-variable "always_get_index_html_lambda" {
-  type    = "string"
-  default = "AlwaysRequestIndexHTML"
-  description = "The lambda that always get index.html for static sites"
-}
+// lambda to associate with a CloudFront distribution
+variable "lambda_arn" {
```

Comment:
Close!  But we need more than one lambda.  We need one for `origin-request` and one for `origin-response` (two separate variables).